src/Microsoft.NetCore.Analyzers/UnitTests/Runtime/SpecifyIFormatProviderTests.cs

```diff
@@ -859,6 +859,50 @@ End Sub
 End Class");
         }
 
+        [Fact]
+        [WorkItem(2394, "https://github.com/dotnet/roslyn-analyzers/issues/2394")]
+        public void CA1305_BoolToString_NoDiagnostics()
```

Comment:
💡 Can you add a test for `string` inputs as well?

Reply:
Good catch! This does raises an issue. I will also add a check for `string`.